src/main/java/sirius/biz/jobs/params/StringParameter.java

```diff
@@ -48,6 +48,12 @@ protected String checkAndTransformValue(Value input) {
         if (input.getRawString() == null) {
             return defaultValue;
         }
+
+        // Always return null when no default value was used and the input is empty or null.
+        if (input.isEmptyString()) {
```

Comment:
In the comment, you say: "if the input is null" ... Can `input` itself be `null` here hence causing a NPE, or did you mean the payload of it?

Reply:
Bad choice of words 😔 
input as _user input_, and not input as the actual variable (which is a Value object and in this context never null)